src/ui/ResultLink/ResultLink.ts

```diff
@@ -500,9 +500,10 @@ export class ResultLink extends Component {
       if (documentURL == undefined || documentURL == '') {
         documentURL = this.escapedClickUri;
       }
-      this.logDocumentOpen(documentURL);
       if (this.options.logAnalytics) {
         this.options.logAnalytics(documentURL);
+      } else {
+        this.logDocumentOpen(documentURL);
```

Comment:
Isn't this reverting the change in the [most recent PR](https://github.com/coveo/search-ui/commit/e934670b30398429bfd5083fa4d0fe79e2e3197b#diff-8c59572edbb1c0c5512df68a2ee098c647558f0351afb0efd9310f8a4adcf389L506), back to what it was before that?
Hmm, was the goal of the last PR to only change `custom` to `click`?

Reply:
Yes. I misunderstood Lucille's suggestion that we should still log a click event as "we should log both click events".